### ISSUE TITLE
MIPI for High Speed?

### ISSUE THREAD

**Issue by mintisan** [m3y54m/Embedded-Engineering-Roadmap#15]
Just as below.

MIPI-DSI and MIPI-CSI is so popular now

![image](https://github.com/m3y54m/Embedded-Engineering-Roadmap/assets/9136049/cbd84b78-607b-43e9-af03-874845181b3d)

**Comment by m3y54m** [comment]
Yes, I agree with you. However they are already in the readme doc of the roadmap. I should make more room to put them on the graphical roadmap.

Anyway if I wanted to add DSI and CSI protocols to the roadmap, I'd rather add a distinc category for them as "Display / Camera" or sth like this.

What do you think?

**Comment by llisauskas** [comment]
> Anyway if I wanted to add DSI and CSI protocols to the roadmap, I'd rather add a distinc category for them as "Display / Camera" or sth like this.

A distinct category would certainly be more appropriate. There is a whole lot of things related to displays and cameras - digital and analog signaling, color spaces, various interfaces, codecs, tools and so on.
